index.js

```diff
@@ -40,10 +44,38 @@ function svg2ttf(svgString, options) {
     throw new Error('svg2ttf: invalid option, version - "' + options.version + '"');
   }
 
-  versionString = 'Version ' + versionString.match(VERSION_RE)[2];
+  var fontVersion = versionString.match(VERSION_RE);
+
+  // Like SourceSansPro's Version 1.095;hotconv 1.0.109;makeotfexe 2.5.65596
+  // or 10.006 beta1
+  versionString = 'Version ' + fontVersion[2];
+
+  // We must update the fontRevision field and ensure that a specific string is
+  // supported. See also: https://silnrsi.github.io/FDBP/en-US/Versioning.html
+  font.revision = fontVersion[3]; // Only number
+
   font.sfntNames.push({ id: 5, value: versionString }); // version ID for TTF name table
   font.sfntNames.push({ id: 6, value: (options.fullname || svgFont.id).replace(/[\s\(\)\[\]<>%\/]/g, '').substr(0, 62) }); // Postscript name for the font, required for OSX Font Book
```

Comment:
In name id 5, the minor version should always be three digits and should refer to the UFO specification（openTypeNameVersion）: https://unifiedfontobject.org/versions/ufo3/fontinfo.plist/#opentype-name-table-fields，I can also consider adding openTypeNameUniqueID

Reply:
In the implementation of ufo2ft, the default version is always 0.000. Use Python's `.zfill(3)` to fill 0 in front of the minor version, similar to `.padStart(3, '0')` in JS.

https://github.com/googlefonts/ufo2ft/blob/742b3a54408dbba99f24a9d21e32b0c499ca6ffe/Lib/ufo2ft/fontInfoData.py#L169-L175